frameioclient/download.py

```diff
@@ -1,18 +1,27 @@
 import requests
-import math
 import os
 
 class FrameioDownloader(object):
-  def __init__(self, asset, download_folder):
+  def __init__(self, asset, download_folder, replace):
     self.asset = asset
     self.download_folder = download_folder
+    self.replace = replace
 
   def download(self):
     original_filename = self.asset['name']
     final_destination = os.path.join(self.download_folder, original_filename)
-    
+
+    if os.path.isfile(final_destination) and not self.replace:
+      try:
+        raise FileExistsError   # Added in python 3.3
```

Comment:
Is there a backwards compatible way of doing this in <3.3 ?

Reply:
Yeah, OSError is the closest match I found. So in < 3.3 it will Name error fail and throw OS instead.

Reply:
Awesome 👍